### DISCUSSION TITLE
테스트 버튼을 눌러도 반응이 없습니다.

### DISCUSSION THREAD

**Discussion by narnarman** [naver/ngrinder#819] [Q&A]
첫 테스트를 시작하고 몇초후에 컴퓨터가 프리징이 걸리더니 윈도우 재부팅을하고 나서 아예 테스트 버튼이 반응을 하지 않습니다. 혹시 해결 방법이 있을까요?

**Comment by donggyu04** [comment]
무슨 상황인지 말씀만 듣고는 모르겠습니다.
저는 해당 상황은 전혀 경험 해본적이 없어서요.

테스트 버튼을 눌렀을때, 서버 로그나, 크롬 개발자 도구 등에 어떤 메시지가 없었나요?
`{사용자 홈}/.ngrinder` 폴더를 삭제하거나 다른 이름으로 백업해 놓으시고 아래의 절차에 따라 한번 해보시기 바랍니다.

1. `{사용자 홈}/.ngrinder` 삭제 or 다른 이름으로 변경
2. [최신 컨트롤러 war 다운로드](https://github.com/naver/ngrinder/releases/download/ngrinder-3.5.5-p1-20210531/ngrinder-controller-3.5.5-p1.war)
3. `java -jar ngrinder-controller-3.5.5-p1.war` 실행 ( JDK 1.8로 하길 권장 드림, JDK11도 가능)
4. 로그인 후 agent 재 다운로드 및 가동
5. 기본 스크립트를 vuser 1 로 실행하여 동작 여부 확인

**Comment by donggyu04** [comment]
도커 실행 환경을 구축하시고, [ngrinder 도커 이미지](https://hub.docker.com/r/ngrinder/controller)를 활용하여 사용하실 수도 있습니다.
